<ui version="4.0" >
 <class>AWatchDialogClass</class>
 <widget class="QDialog" name="AWatchDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>347</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Watch Variable List</string>
  </property>
  <widget class="QTreeView" name="upVarListTree" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="editTriggers" >
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="sortingEnabled" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>300</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <item>
     <widget class="QPushButton" name="upAddButton" >
      <property name="text" >
       <string>Add Variable</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="upRemoveButton" >
      <property name="text" >
       <string>Remove Variable</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="upRemoveAllButton" >
      <property name="text" >
       <string>Remove All</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="upCloseButton" >
      <property name="text" >
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
